src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -105,7 +109,7 @@
       <Link>Common\System\HResults.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'netcoreapp1.2corert'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'netcoreapp1.2corert'">
```

Comment:
yeah, there are only a few libraries that had this configuration, so once I'm done with uap and uapaot verticals I will move on to netcoreapp1.2corert. cc: @SedarG